extensions/gitlab/src/gitlabapi.ts

```diff
@@ -98,7 +98,7 @@ export function jsonDataToMergeRequest(mr: any): MergeRequest {
     updated_at: mr.updated_at,
     author: maybeUserFromJson(mr.author),
     assignees: mr.assignees.map(userFromJson),
-    reviewers: mr.reviewers.map(userFromJson),
+    reviewers: (mr.reviewers && mr.reviewers.map(userFromJson)) || [],
```

Comment:
```suggestion
    reviewers: mr.reviewers?.map(userFromJson) || [],
```

Same code, but nicer to read 🙂

Reply:
👍